src/server_manager/messages/master_messages.json

```diff
@@ -319,6 +319,14 @@
     "message": "Failed to get list of available regions",
     "description": "This string appears in an error notification toast. It is shown when there is an error retrieving the regions available for server deployment."
   },
+  "error_do_limit": {
+    "message": "Your DigitalOcean account has reached its Droplet limit.  You can request an increase at https://cloud.digitalocean.com/account/team/droplet_limit_increase",
+    "description": "This string appears in an error notification toast. It is shown when the user has created the maximum number of allowed servers."
+  },
+  "error_do_warning": {
+    "message": "DigitalOcean warning:",
```

Comment:
I think we should make the warning a parameter of the message.
Then you can add quotes. I also wonder what happens with RTL

Reply:
Done